tox.ini

```diff
@@ -7,12 +7,11 @@ skipsdist = true
 envlist = modification,py3-cover,lint,mypy
 
 [base]
-# pip installs the requested packages in editable mode
-pip_install = python {toxinidir}/tools/pip_install_editable.py
```

Comment:
I think we should move `pip_install` before the comment intended for `install_and_test` and keep the comment written for `pip_install`.

Reply:
Done.